<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtzPrivacyMaskPage</class>
 <widget class="QWidget" name="PtzPrivacyMaskPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_content" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>18</number>
      </property>
      <item row="0" column="0">
       <widget class="CommonVideo" name="video" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PtzControlPanel" name="ptzControlPanel" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_maskContent" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0">
            <widget class="QLabel" name="label_region">
             <property name="text">
              <string>Region</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelRegionType">
             <property name="text">
              <string>Region Type</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="MyPushButton" name="pushButton_add">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyPushButton" name="pushButton_clear">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyPushButton" name="pushButton_clearAll">
                <property name="text">
                 <string>Clear All</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_enable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>PTZ Privacy Mask</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="MyCheckBox" name="checkBox_enable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelMaskColor">
             <property name="text">
              <string>Mask Color</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ComboBox" name="comboBoxRegionType"/>
           </item>
           <item row="2" column="1">
            <widget class="ComboBox" name="comboBoxMaskColor"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="TableView" name="tableView">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>272</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>272</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BlueLabel" name="label_note">
           <property name="text">
            <string>Note: Support up to 8 Privacy Mask areas.</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ButtonGroup" name="channelGroup" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="WidgetMessage" name="widgetMessage" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header>MyCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>CommonVideo</class>
   <extends>QWidget</extends>
   <header>commonvideo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonGroup</class>
   <extends>QWidget</extends>
   <header>buttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlueLabel</class>
   <extends>QLabel</extends>
   <header>bluelabel.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetMessage</class>
   <extends>QWidget</extends>
   <header>widgetmessage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PtzControlPanel</class>
   <extends>QWidget</extends>
   <header>ptzcontrolpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
